apps/discovery-platform/src/db/index.ts

```diff
@@ -36,24 +39,30 @@ export const getNonExitNodes = async (
 
 export const saveRegisteredNode = async (
   dbInstance: DBInstance,
-  node: QueryRegisteredNode
+  node: Omit<QueryRegisteredNode, "created_at" | "updated_at">
 ): Promise<boolean> => {
   try {
-    const text = `INSERT INTO 
-    ${TABLES.REGISTERED_NODES} (has_exit_node, id, chain_id, total_amount_funded, honesty_score, status)
-    VALUES ($<has_exit_node>, $<id>, $<chain_id>, $<total_amount_funded>, $<honesty_score>, $<status>)
+    const text = `INSERT INTO
+    ${TABLES.REGISTERED_NODES} (has_exit_node, id, chain_id, hoprd_api_endpoint, hoprd_api_port, exit_node_pub_key, node_address, total_amount_funded, honesty_score, status)
+    VALUES ($<has_exit_node>, $<id>, $<chain_id>, $<hoprd_api_endpoint>, $<hoprd_api_port>, $<exit_node_pub_key>, $<node_address>, $<total_amount_funded>, $<honesty_score>, $<status>)
```

Comment:
lets call it `native_address` so it matches the one used by the API, `native_address` = ETH / xDAI address